core/src/test/java/org/apache/calcite/test/RelOptRulesTest.java

```diff
@@ -6562,6 +6562,67 @@ private HepProgram getTransitiveProgram() {
         .check();
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-6214">[CALCITE-6214]
+   * Remove `DISTINCT` in `COUNT` if field is unique</a>. */
```

Comment:
Don't use backticks in jira summary or javadoc. They will be rendered as backticks.

Reply:
Removed them.